src/modules/RootPart.rb

```diff
@@ -1806,7 +1806,7 @@ def MountPartitions(root_device_current)
         # in the target map of the storage module
         AddToTargetMap()
         if Yast2::FsSnapshot.configured?
-          snapshot = Yast2::FsSnapshot.create_pre("before update", cleanup: :number, important: true)
+          snapshot = Yast2::FsSnapshot.create_pre(_("before update"), cleanup: :number, important: true)
```

Comment:
Please add comment for translators - vitally important for good translation as the text itself does not say anything.